content/posts/2024-08-29-GSoC-2024-statistical_analysis_of_llvm-ir_compilation.md

```diff
@@ -0,0 +1,101 @@
+---
+author: "Andrew Kallai"
+date: "2024-08-29"
+tags: ["GSoC", "llvm-ir-dataset-utils", "ComPile", "analysis"]
+title: "GSoC 2024: Statistical Analysis of LLVM-IR Compilation"
+---
+
+Welcome! My name is Andrew and I contributed to LLVM through the 2024 Google Summer of Code Program. My project is called [Statistical Analysis of LLVM-IR Compilation](https://summerofcode.withgoogle.com/programs/2024/projects/hquDyVBK). The objective of this project is to provide an analysis of the causes of unexpected compilation times.
```

Comment:
I would maybe reword the last couple words. Unexpected compilation times is kind of vague. Maybe something to the tune of "an analysis of where time spent in the optimization pipeline drastically increases in some instances and why this occurs".